src/cachemachine/rubinrepoman.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, List, Set
+
+import structlog
+from docker_registry_client import DockerRegistryClient
+
+from cachemachine.dockercreds import DockerCreds
+
+logger = structlog.get_logger(__name__)
+
+
+class RubinRepoMan:
+    registry_url: str
+    recommended_image_url: str
+    recommended_tag: str
+    num_dailies: int
+    num_weeklies: int
+    num_releases: int
+
+    def __init__(self, body: Dict[str, Any]):
+        self.registry_url = body.get("registry_url", "hub.docker.com")
+        (self.username, self.password) = DockerCreds.lookup(self.registry_url)
+        self.repo = body["repo"]
+
+        self.recommended_image_url = body.get("recommended_image_url", " : ")
```

Comment:
This is an inobvious default value. I guess it's intended to set `self.recommended_tag` to a single space? But why that value?